### PR TITLE
update instructions for running

### PR DESCRIPTION
**Pull request by evanshortiss**
The current commands for running the UPS are missing some details that cause the startup to fail. 

These updated commands work nicely, but might need some changes that you folks are aware of.

I also added a little more detail about accessing the UPS so people can get started a little faster.

Thank you @shannon for the `--net` protip

### CODE REVIEW

File: wildfly/unifiedpush-wildfly/README.md [hunk 1]
```diff
@@ -18,45 +18,48 @@ Follow the [instructions](http://docs.docker.com/installation/)
...
-
-For the database of the UnifiedPush Server itself, a similar command is needed:
+You can start a database for the UnifiedPush Server using the following command:
 
 ```shell
-docker run --name unifiedpushDB \
+$ docker run --name unifiedpushDB \
            -p 11306:3306 \
+           -p 8080:8080 \
+           -p 9090:9090 \
```
Comment (matzew) [inline wildfly/unifiedpush-wildfly/README.md]: why so many port mappings ?

Comment (matzew) [reply wildfly/unifiedpush-wildfly/README.md]: do we need the mgmt port ?

Comment (evanshortiss) [reply wildfly/unifiedpush-wildfly/README.md]: @matzew we need to expose 8080 for Keycloak access and 9090 for UPS. I think the MySQL port can be removed, so I'll check that.

**Comment by evanshortiss** [comment]
@matzew db no longer exposed 👍

**Comment by evanshortiss** [comment]
@matzew no problem, think it should be good now.

**Comment by matzew** [comment]
@evanshortiss thanks :tada:

**Pull request merged**